store_redis.go

```diff
@@ -7,21 +7,59 @@ import (
 	"github.com/garyburd/redigo/redis"
 )
 
+// RedisStoreFunc is a redis store function.
+type RedisStoreFunc func(c redis.Conn, key string, rate Rate) ([]int, error)
+
+// RedisStoreOptions are options for Redis store.
+type RedisStoreOptions struct {
+	// The prefix to use for the key.
+	Prefix string
+
+	// The maximum number of retry under race conditions.
+	MaxRetry int
+}
```

Comment:
Remove the ``Redis`` prefix, we will share ``StoreOptions`` between memory and redis